code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -214,6 +214,16 @@
 	for(var/i=0,i<5,i++)
 		new /obj/item/weapon/grenade/empgrenade(src)
 
+/obj/item/weapon/storage/box/wind
+	name = "wind grenades"
+	desc = "A box containing 3 wind grenades"
+	icon_state = "flashbang"
+
+/obj/item/weapon/storage/box/wind/New()
+	..()
+	for(var/i = 0, i < 3, i++)
```

Comment:
good catch, thanks

That's what happens when you lazily copy down loops. I know it's not atomic, but I might just go ahead and fix all the loops in the file so that this doesn't happen to anyone again